Resource.rc

```diff
@@ -93,18 +93,27 @@ BEGIN
     LTEXT           "Pattern Change Frequency:",IDC_STATIC,228,199,90,8
     LTEXT           "Often",IDC_STATIC,228,217,20,8
     LTEXT           "Rarely",IDC_STATIC,360,217,22,8
-    PUSHBUTTON      "Defaults",IDC_DEFAULTS_BUTTON,222,295,50,14
+    PUSHBUTTON      "Defaults",IDC_DEFAULTS_BUTTON,222,379,50,14
     COMBOBOX        IDC_YONK_PATTERN,228,133,78,30,CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT           "< https://doughbyte.com >",IDC_STATIC,222,277,90,8,WS_DISABLED
+    LTEXT           "< https://doughbyte.com >",IDC_STATIC,222,361,90,8,WS_DISABLED
     CONTROL         "Do not change",IDC_PATTERN_FIX,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,328,199,63,10
     LTEXT           "Population Diversity:",IDC_STATIC,11,18,67,8
     LTEXT           "None",IDC_STATIC,12,78,18,8
     LTEXT           "All",IDC_STATIC,156,78,8,8
     CONTROL         "",IDC_IMPOSTOR_CHANCE,"msctls_trackbar32",TBS_BOTH | WS_TABSTOP,48,72,102,24
     LTEXT           "Palettes:",IDC_STATIC,203,60,26,8
     COMBOBOX        IDC_YONK_PALETTE,203,72,78,30,CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT           "yokscr version ???",IDC_VERSION_LABEL,222,264,149,8,WS_DISABLED
+    LTEXT           "yokscr version ???",IDC_VERSION_LABEL,222,348,149,8,WS_DISABLED
     CONTROL         "Play over the desktop",IDC_PLAY_OVER_DESKTOP,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,228,240,158,10
+    LTEXT           "Trail Length:",IDC_STATIC,12,289,41,8
+    CONTROL         "",IDC_TRAIL_LENGTH,"msctls_trackbar32",TBS_BOTH | WS_TABSTOP,48,301,100,24
+    LTEXT           "Short",IDC_STATIC,12,307,18,8
+    LTEXT           "Long",IDC_STATIC,155,307,16,8
+    LTEXT           "Trail Spacing:",IDC_STATIC,12,331,44,8
+    LTEXT           "Near",IDC_STATIC,12,349,16,8
+    CONTROL         "",IDC_TRAIL_SPACE,"msctls_trackbar32",TBS_BOTH | WS_TABSTOP,48,343,102,24
+    LTEXT           "Distant",IDC_STATIC,155,349,24,8
+    CONTROL         "Enable trails",IDC_TRAILS_ENABLED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,132,289,55,10

```

Comment:
![image](https://github.com/pierce-smith1/yok.scr/assets/50504215/d05c214d-4912-4eee-b47d-4b80f042461b)

UX: If trails are disabled, the sliders should be disabled too. Letting people twiddle sliders that do nothing is really confusing. Look at the pattern fixed checkbox and the pattern duration slider to tell how I did this in the past.